templates/shared/contextual_footers/_cloud_bootstack.html

```diff
@@ -1,5 +1,5 @@
 <div class="col-4 p-divider__block">
   <h3 class="p-heading--four">Want a fully managed private cloud?</h3>
   <p>BootStack is your OpenStack private cloud with our experts responsible for design, deployment and availability.</p>
-  <p><a href="/cloud/openstack/managed-cloud/contact-us" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'BootStack', 'eventLabel' : 'Want a fully managed private cloud?', 'eventValue' : undefined });">Get BootStack&nbsp;&rsaquo;</a></p>
+  <p><a href="/cloud/contact-us?product=contextual-footer-managed-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Contextual footer link', 'eventAction' : 'BootStack', 'eventLabel' : 'Want a fully managed private cloud?', 'eventValue' : undefined });">Get BootStack&nbsp;&rsaquo;</a></p>
```

Comment:
Shouldn't this just be `managed-cloud` as there is no option for `contextual-footer-managed-cloud`.